From 464acc7b9dc21257ae196bdc1618188041de523b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matthew=20=F0=9F=AA=B7?= <matthew@urgero.net>
Date: Sat, 10 May 2025 12:47:13 -0500
Subject: [PATCH 23/46] Serious bump version to 9.4.0 (#19)

Mrs. Turtle reverted versioning back to 8.3.x, I must have forgotten to
merge one of the files in a previous commit
---
 CMakeLists.txt             | 2 +-
 contrib/aur/lotus/PKGBUILD | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c4e8f854a..c13862fef 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,7 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
 )
 
 project(lotus
-	VERSION 8.3.27
+	VERSION 9.4.0
 	DESCRIPTION "Lotus is a full node implementation of the Lotus protocol."
 	HOMEPAGE_URL "https://lotusia.org"
 )
diff --git a/contrib/aur/lotus/PKGBUILD b/contrib/aur/lotus/PKGBUILD
index 50e16467e..cd82c8613 100644
--- a/contrib/aur/lotus/PKGBUILD
+++ b/contrib/aur/lotus/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Josh Ellithorpe <quest@mac.com>
 
 pkgname=lotusd
-pkgver=8.3.27
+pkgver=9.4.0
 pkgrel=0
 pkgdesc="Lotus with lotusd, lotus-tx, lotus-seeder and lotus-cli"
 arch=('i686' 'x86_64')
-- 
2.39.5 (Apple Git-154)

